### PR TITLE
Worked on react and some practice questions

### PR DESCRIPTION
**Pull request by Akanksha987**
->Solved "Smallest positive missing number" using python
->Solved "Valid Anagram " using python 
->Worked upon Signup and Login page in react

### CODE REVIEW

File: client/src/components/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      ),
+  });
+
+  const onSubmit = (values, { setSubmitting }) => {
+    setTimeout(() => {
+      console.log(values);
+      sessionStorage.setItem("name", values.name);
+
+      setSubmitting(false);
+      window.open("http://localhost:3000/", "_self");
```
Comment (PatelNeelMahesh) [inline client/src/components/Login/Login.js:33]: `http://localhost:3000/` -> process.env.Domain

File: client/src/components/Signup/Signup.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      .required("Please confirm the password"),
+  });
+
+  const onSubmit = (values, { setSubmitting }) => {
+    setTimeout(() => {
+      console.log(values);
+      sessionStorage.setItem("name", values.name);
+
+      setSubmitting(false);
+      window.open("http://localhost:3000/", "_self");
```
Comment (PatelNeelMahesh) [inline client/src/components/Signup/Signup.js:40]: `http://localhost:3000/` -> process.env.Domain

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]

File: client/src/components/Signup/Signup.js (hunk 2 above)
Comment (Akanksha987) [reply client/src/components/Signup/Signup.js:40]: Sure, I will do it in next PR

File: client/src/components/Login/Login.js (hunk 1 above)
Comment (Akanksha987) [reply client/src/components/Login/Login.js:33]: Sure, I will do it in next PR

**Pull request merged**